### ISSUE TITLE
Resolve multiple address/instruction listings for same hospital

### ISSUE THREAD

**Issue by cchang-adi** [r-pop/findthemasks#233]
As we begin to collect more data, we are increasingly seeing multiple listings for hospitals that aren't exact duplicates, submitted over the course of several days/hours. They roughly fall into this pattern:

1) Hospital X, official address, official contact/links (either hospital-wide or specific dept)
2) Hospital X, nearby residential address (almost certainly staff), personal phone # or "leave with doorman"
3) Hospital X, official address, different specific department from listed in 1)
4) Hospital X, official address, yet another department with contact info different from 1) and 3)

They do not seem to be clear updates to each other, but rather separate requests for PPE from different departments or individuals. In general we try to post every request, so we've put these up. We have tried to merge these in the past, which leads to "Instructions" field becoming a multi-line block with line breaks like row 48:

"1 
OR 
2 
OR 
3
OR 
4"

This is what it looks like on the web: 
![image](https://user-images.githubusercontent.com/45407457/77679991-8f45bb00-6f50-11ea-9494-6da0ada7765b.png)

This seems unwieldy and unreadable. Any good UX or format ideas for this situation?

If it helps we can add a new dedupe flag to indicate that this isn't really a dupe, just a variant.

**Comment by marcuscollins** [comment]
Seems like a schema problem. Can we add a department field (or a "care of" field?) Then we we can group by facility (hospital/clinic) and list the needs by dept. or c/o under the facility?

**Comment by cchang-adi** [comment]
I agree, we should include the option for us to split a single hospital listing into multiple departments or sub-bullet points of some kind. I've already asked for line breaks in the raw-data irnstructions field to be honored, so it seems like splitting each request into its own line is a start, although a better solution would be a clearly formatted department field of some sort.

**Comment by r-pop** [comment]
Hi @cchang-adi is this still accurate? I know we did some stuff with dual pins on same location - is this good enough or if not can you restate what the next task is?

**Comment by cchang-adi** [comment]
The dual pins problem fixed locations whose markers were right next to each other and were impossible to click independently (ie a health system in AZ with two different clinics, in two suites side by side). 

This problem is more to help clean up the “Instructions” field in listings that have you to four (!) unique sets of instructions. This is a worst case example of what it looks like now. 

![CAE1501C-38A0-49BD-981D-592E6B82F3AF](https://user-images.githubusercontent.com/45407457/80933065-ed976200-8d76-11ea-8797-64189bfbd0dd.png)
![C0E5C9F9-1C38-4A94-B641-A3F5861BFBE0](https://user-images.githubusercontent.com/45407457/80933067-eff9bc00-8d76-11ea-8b1b-adf565a5ed1c.png)

There are so many instructions that the pin cuts it off. The very last department might be at a disadvantage too. And it just takes up a lot of space with these ORs, when using another level of separation is really what’s needed.
